src/components/Navigation/Navbar.tsx

```diff
@@ -24,7 +24,7 @@ export function Navbar() {
       component: <UploadImage />,
     },
     {
-      route: "user/main/creatives",
```

Comment:
We should add in a (client-side) redirect from the old url, just as a matter of principle.